src/main/java/org/jenkinsci/plugins/xunit/threshold/XUnitThreshold.java

```diff
@@ -94,27 +94,28 @@ public Result getResultThresholdNumber(XUnitLog log,
                                            int testCount,
                                            int newTestCount) {
 
+        String errorMessage = "The total number of tests for this category exceeds the specified '%s' threshold value.";
         if (isValid(getFailureThreshold())
                 && (convertToInteger(getFailureThreshold()) < testCount)) {
-            log.infoConsoleLogger("The total number of tests for this category exceeds the specified 'failure' threshold value.");
+            log.infoConsoleLogger(String.format(errorMessage, "failure"));
             return Result.FAILURE;
         }
 
         if (isValid(getFailureNewThreshold())
                 && (convertToInteger(getFailureNewThreshold()) < newTestCount)) {
-            log.infoConsoleLogger("The new number of tests for this category exceeds the specified 'new failure' threshold value.");
```

Comment:
I wonder whether this cosmetic change might affect some people if they use their console log output to be parsed in order to do something else, for instances if they use the groovy post build action. Probably it's worth to add that information as part of the release notes